revolv/templates/base/animations/nav_spine.html

```diff
@@ -28,4 +29,37 @@
             <figcaption class="third-caption">PAY IT FORWARD</figcaption>
             Finally, we reinvest the money saved to fund more worthy solar energy projects.
         </figure>
-</div>
\ No newline at end of file
+</div>
+{% else %}
+<div 
+    data-300="margin-left: -100px;"
+    data-700="margin-left: 100px;"
+    data-1750="margin-left: 100px;"
+    data-1850="margin-left: -100px;"
+    class="nav-spine">
+        <div class="first nav-spine-item">1</div>
```

Comment:
The actual text here is duplicated from above, even though the offsets are different. I think it would be good to have every offset group in a specific if block by itself: this way, even though there's a lot of if/else logic, we're not duplicating strings, so the duplication only happens where it needs to.